Example/TouchID/ViewController.swift

```diff
@@ -17,8 +17,36 @@ public final class ViewController: UIViewController {
 
     // MARK: - Private properties -
 
+    @IBOutlet private weak var storeResultLabel: UILabel!
+    @IBOutlet private weak var readResultLabel: UILabel!
+
     private let identifier = "TouchIDSampleApp"
 
+    public override func viewDidLoad() {
+        super.viewDidLoad()
+//        Locker.setKeychainService("someCustomService")
+//        shouldUseAuthWithBiometrics = true
```

Comment:
I intended to leave it commented here as an example of how and where you can set a custom service and toggle the biometry usage.

Reply:
I would leave it uncommented with some comments before it:
```suggestion
         // Optional configurations
         // If needed, custom service can be set. Otherwise, Locker will set one for you.
        Locker.setKeychainService("someCustomService")
         // Define if biometrics should be used. If not, the device PIN will be prompted
        shouldUseAuthWithBiometrics = true
```

On that note, should we give an option for disabling the biometrics? 
The sole purpose of Locker was to create a wrapper that would use a keychain behind the biometrics lock. Our logo is created that way, and a general description is set behind that idea:
> Lightweight library for handling sensitive data (String type) in Keychain using iOS Biometric features

If we give an option not to use biometrics, then we should consider changing the description.

Reply:
I get your point, I see Locker as a wrapper around Keychain, i.e. makes it easy to use Keychain, whether you choose to lock it biometrically or not. So maybe the new tagline should just read:
> Lightweight library for handling sensitive data (String type) in Keychain